content/en/Cobalt-API/_index.md

```diff
@@ -18,13 +18,13 @@ The Cobalt [RESTful API](/getting-started/glossary/#restful-api) gives you acces
 
 Here are some examples of how you can scale your workflows:
 
-- Retrieve [findings](https://docs.cobalt.io/v2/#findings) that our pentesters discovered during a pentest.
```

Comment:
Does the docs framework have any kind of functionality that would let us pass the section IDs (ex: `#findings`) from the parent page URL to the iframe URL? Can we run custom javascript within the docs page?